src/tests/test-utils.tsx

```diff
@@ -30,30 +31,42 @@ const mockRouter = (props: Partial<NextRouter> = {}): NextRouter => ({
 
 // Add in any providers here if necessary:
 // (ReduxProvider, ThemeProvider, etc)
-const getProviders: (routerProps: Partial<NextRouter>) => React.FC<{ children: React.ReactElement }> = (routerProps) =>
+const getProviders: (options: {
+  routerProps?: Partial<NextRouter>
+  initialReduxState?: Partial<RootState>
+}) => React.FC<{ children: React.ReactElement }> = ({ routerProps, initialReduxState }) =>
   function ProviderComponent({ children }) {
     const { StoreHydrator } = require('@/store') // require dynamically to reset the store
 
     return (
-      <StoreHydrator>
+      <StoreHydrator initialState={initialReduxState}>
```

Comment:
I would mock localStorage instead. That way we would also test that hydration from LS works.

Reply:
It still seems to work because a couple of tests depend on local storage persistence. I'm not sure if hydration is relevant here because, AFAIK, we plan to export the app to a static HTML

Reply:
"hydration" related to store population here. It's not related to NextJS hydration.